jest.config.js

```diff
@@ -4,7 +4,13 @@ module.exports = {
   clearMocks: true,
   collectCoverage: true,
   coverageThreshold: {
-    global: {branches: 100, functions: 100, lines: 100, statements: 100},
+    'global': {branches: 100, functions: 100, lines: 100, statements: 100},
+    './src/cdk/utils/basic-authorizer-handler/index.ts': {
+      branches: 33,
```

Comment:
Coverage measurement should be completely disabled for the file. These values are fragile and of no use. Once the file is touched, Jest would fail.

Reply:
You could use `coveragePathIgnorePatterns: 'src/cdk/utils/basic-authorizer-handler/index.ts'`

Reply:
The custom coverage values would ensure that at least the coverage won't decrease. But yes, these values are fragile...

Anyway, 100% coverage seems to be unrealistic and would make contributions difficult or it will end up in adding more files to `coveragePathIgnorePatterns` which should not be the goal. (In my opinion pull requests should come with unit tests for the new or modified functions, but the contributor shouldn't need to write tests for the rest of the file.)

Custom coverage values changed to `coveragePathIgnorePatterns`.

Reply:
I think the problem only exists in this specific case. Since we do not bundle the Lambda function, we cannot outsource functions. Normally we would have packed this function into an external file and tested it separately. But maybe you are right and I should switch off the coverage completely. The code base is only partially tested anyway and yes it could prevent contributions, make it unnecessarily difficult.